docker/ci/build-image-multi-arch.sh

```diff
@@ -46,12 +47,15 @@ function cleanup_all() {
     cleanup_docker_buildx
     File_Delete $DIR
 }
-while getopts ":hv:f:" arg; do
+while getopts ":hr:v:f:" arg; do
     case $arg in
         h)
             usage
             exit 1
             ;;
+        r)
```

Comment:
Are we gonna input this `REPO_NAME` from the jenkins build script command in the parameter when running `docker_build`?

Reply:
No, that one is using the scripts in docker/release dir, not in docker/ci dir like in this PR.

Reply:
Just to make sure, with this change, we would need to add `REPO_NAME` here? https://github.com/opensearch-project/opensearch-build/blob/31991db16bd1d83bfc0e380a9c130994b9e40314/jenkins/docker/docker-build.jenkinsfile#L19-L24

Reply:
I can tweak the description of this, and the defaultvalue.

Reply:
Resolved.